templates/api.html

```diff
@@ -40,7 +40,7 @@
       CDNJS
     </a>(except auto-update config, we'll only return the auto-update type (currently <code>npm or git</code>)
     <br><br>
-    The API will return the maximum results by default (1000), but you can limit these the "hitsPerPage" query
+    The API will return the maximum results by default (1000), but you can limit these with the "hitsPerPage" query
```

Comment:
@tholman Wow, you're quick! Just realised I commented in the wrong place and wanted to do it here, but you've already fixed it, amazing! :tada: